src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -431,15 +431,29 @@ def build_cmd(self, env, host, **kwargs):
 
         return tst_cmd
 
-    def convert_xml(self, xml_file):
-        """Modify the xml file"""
+    def _rm_abs_src_path_from_xml(self, xml_file):
```

Comment:
You can avoid this warning  by making this a staticmethod, as below.
```suggestion
    @staticmethoc
    def _rm_abs_src_path_from_xml(xml_file):
```